src/app/components/play-round/play-round.component.html

```diff
@@ -17,13 +17,15 @@ <h1 class="nest-count">{{(this.studentRoundService.currentNestPollen$ | async) }
   [ngStyle]="{'transform': 'translate(-50%, 25%) scale(' + calculateBeeScale(bee.relative_size) + ')'}"
   [alt]="bee.name"
 >
-<button
-  mat-raised-button
-  class="interact-button"
-  color="primary"
-  *ngIf="this.foundRoundMarker$ | async as marker"
-  [disabled]="!marker.active || ((this.studentRoundService.currentBeePollen$ | async) >= 3 && !marker.isNest)"
-  (click)="clickInteract(marker)"
->
-  Visit {{marker.name}}
-</button>
+<ng-container *ngIf="{ count: (this.studentRoundService.currentBeePollen$ | async) } as currentBeePollen">
```

Comment:
Why is this being made an object? Could it not just be this:
```suggestion
<ng-container *ngIf="(this.studentRoundService.currentBeePollen$ | async) as currentBeePollen">
```
(and change the reference below to remove `.count`)

Reply:
oh wait I see it's so when the current pollen is 0 it doesn't evaluate as false. Seems kinda hacky though.